### ISSUE TITLE
Show a notice about how browser-laptop is deprecated

### ISSUE THREAD

**Issue by bsclifton** [brave/brave-browser#1544]
~Blocked on https://github.com/brave/brave-browser/issues/1542~

This might be an about page that gets shown on launch?
`about:deprecated` or similar

We could show it once and only re-show it if folks use the menu and `Check for update`

For all platforms, we could basically show a few things:
- brave-core is the new version; it should be installed. If not, we could potentially have a button to re-install
- we can share that this browser will continue to work, but that it won't receive updates
- Brave Payments users are encouraged to go to their ledger and save PDFs for any payments they may care about

For Linux, we could display instructions on how to modify `/etc/repos.d/` or just link to the new install guide

**Comment by bsclifton** [comment]
cc: @bradleyrichter for an example that we can show

Perhaps we can replace the welcome page? (and show it every launch)

**Comment by bsclifton** [comment]
Example proposed by @bradleyrichter:
![image](https://user-images.githubusercontent.com/4733304/47584167-4e3a1900-d90e-11e8-8887-18bf45ea3f0b.png)

**Comment by srirambv** [comment]
@bsclifton @bradleyrichter Will the message be shown only once or on each new tab? Should probably made persistent and not closed by user

**Comment by bsclifton** [comment]
@srirambv I agree- should always show and not have an X in the top right

**Comment by bsclifton** [comment]
Starting dev on this

**Comment by bsclifton** [comment]
Initial implementation:
<img width="1506" alt="screen shot 2018-10-29 at 12 22 34 pm" src="https://user-images.githubusercontent.com/4733304/47675863-50071500-db78-11e8-8684-e01f278593b6.png">

Need help getting a support URL to link to in the "Learn more..."

**Comment by davidtemkin** [comment]
@bsclifton @bradleyrichter This looks good. However I would recommend linking directly to the Brave download page (or even to the download itself). Can have "learn more" as a subsidiary link for people who are not ready to simply download the new browser.

**Comment by bsclifton** [comment]
As shared in the demo today, this is AFTER the update has been downloaded (by this time, brave-core should be installed)

Will re-share the above screenshot on Slack to make sure we are covered on all platforms. I believe @diracdeltas may want to add additional messaging for Linux users

**Comment by diracdeltas** [comment]
The linux message should be (1) more annoying/insistent and (2) shown regardless of whether the user has installed brave-core.

I have opened a separate issue for it: https://github.com/brave/browser-laptop/issues/15311

Triage team, please mark this as appropriate to make sure it's not lost

**Comment by bradleyrichter** [comment]
@bsclifton Here are the updated mocks for Linux vs mac/win.

![image](https://user-images.githubusercontent.com/13509546/47805498-b1ec8980-dcf4-11e8-9c5f-4d04cd55ef66.png)

![image](https://user-images.githubusercontent.com/13509546/47805521-c16bd280-dcf4-11e8-9abe-11ced877d9b2.png)

**Comment by bsclifton** [comment]
Finished and tested implementations on macOS/Windows/Linux; all instances match spec

**Comment by rebron** [comment]
Definitely need uninstall instructions the 'instructions' link and 'learn more' link. We might have too much text in this notification already to add a "by the way, uninstall this browser when you get around to it." It's a 3rd or 4th piece of information we want users to know and could be too much already. 

I'd like to keep this notification upgrade centric.

**Comment by bradleyrichter** [comment]
I think this feels weird: "We recommend uninstalling this browser after you're able to move your data over". I think the mention of security is enough. It's pretty clear that we nudging you to move on and leave this one behind.

**Comment by davidtemkin** [comment]
@bradleyrichter agree, we have been quite clear. uninstall notice is superfluous and should not be added.

**Comment by Brave-Matt** [comment]
@srirambv and I have linux a linux uninstall doc planned that you could link to in the browser, if that's useful.

**Comment by diracdeltas** [comment]
agree that an uninstall notice is probably superfluous. at least on mac/linux, it is pretty common for apps to not handle their own install in the UI, so users are accustomed to having to do it themselves anyway.

**Comment by bsclifton** [comment]
Closing because what we have (per discussion above) should work fine 👍
